libs/CMakeLists.txt

```diff
@@ -59,6 +59,7 @@ set(HPX_CANDIDATE_LIBS
     itt_notify
     lcos_distributed
     lcos_local
+    libcds
```

Comment:
Would `cds` be enough for the module name?

Reply:
I will prefer `libcds` as it sounds like it is an external library and not implemented by hpx.